mountpoint-s3/Cargo.toml

```diff
@@ -7,8 +7,8 @@ publish = false
 
 [dependencies]
 fuser = { path = "../vendor/fuser", features = ["abi-7-28"] }
-mountpoint-s3-client = { path = "../mountpoint-s3-client" }
-mountpoint-s3-crt = { path = "../mountpoint-s3-crt" }
+mountpoint-s3-client = { path = "../mountpoint-s3-client", version = "0.2.1" }
+mountpoint-s3-crt = { path = "../mountpoint-s3-crt", version = "0.2.1" }
```

Comment:
not to change in this PR, but we were wondering why Mountpoint is depending directly on the CRT crate.

Reply:
Looks like it's currently just for `RustLogAdapter`. I guess we could re-export that from `mountpoint-s3-client` but that feels a little weird.